packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -0,0 +1,150 @@
+mod data_type;
+
+use error_stack::Result;
+use graph::{
+    datastore::{
+        DatabaseConnectionInfo, DatabaseType, Datastore, InsertionError, PostgresDatabase,
+        QueryError, UpdateError,
+    },
+    types::{schema::DataType, AccountId, BaseUri, Qualified, VersionId},
+};
+use sqlx::{Connection, PgConnection};
+use tokio::runtime::Runtime;
+use uuid::Uuid;
+
+pub struct Database {
+    postgres: PostgresDatabase,
+    connection: Option<PgConnection>,
+    created_base_uris: Vec<BaseUri>,
+    account_id: AccountId,
+    rt: Runtime,
+}
+
+impl Database {
+    pub fn new() -> Self {
+        const USER: &str = "postgres";
+        const PASSWORD: &str = "postgres";
+        const HOST: &str = "localhost";
+        const PORT: u16 = 5432;
+        const DATABASE: &str = "graph";
+
+        let connection_info = DatabaseConnectionInfo::new(
+            DatabaseType::Postgres,
+            USER.to_owned(),
+            PASSWORD.to_owned(),
+            HOST.to_owned(),
+            PORT,
+            DATABASE.to_owned(),
+        );
+
+        let rt = Runtime::new().expect("Could not create a test runtime");
+        let (postgres, connection, account_id) = rt.block_on(async {
+            let postgres = PostgresDatabase::new(&connection_info)
+                .await
+                .expect("could not connect to database");
+
+            let mut connection = PgConnection::connect(&connection_info.url())
```

Comment:
Why do we need the connection if we have the pool inside `PostgresDatabase`?

Reply:
Can you put a comment saying that

Reply:
Put it in the struct: f3dba9a84ebc49c19666f5944940acb7cfe14c80